### PR TITLE
Makes mentornoot have a 10% chance of playing

### PR DESCRIPTION
**Pull request by adamsong**
# Github documenting your Pull Request

1% chance for mentornoot to play

# Changelog

:cl: 
tweak: tweaked mentornoot to play 10% of the time
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Mentor helps arent jokes

**Review by Bibby0110 — DISMISSED** [review]

**Comment by N3D6** [comment]
> Mentor helps arent jokes

eh i think mentornoot sounds better than the normal honk honestly

**Comment by maxion12345** [comment]
> Mentor helps arent jokes

its a joke

**Comment by alexkar598** [comment]
> > Mentor helps arent jokes
> 
> eh i think mentornoot sounds better than the normal honk honestly

Then replace it entirely that's fine I dont really care but dont make it a 1% chance.

**Comment by boodaliboo** [comment]
> > > Mentor helps arent jokes
> > 
> > 
> > eh i think mentornoot sounds better than the normal honk honestly
> 
> Then replace it entirely that's fine I dont really care but dont make it a 1% chance.

cry harder, oh no theres a funny meme sound that plays 1/100 that most mentors wont even notice or comment on how sad

**Comment by adamsong** [comment]
> Mentor helps arent jokes

Doesn't mean they can't be funny

File: yogstation/code/modules/mentor/mentor.dm [hunk 1]
```diff
@@ -101,12 +101,12 @@ GLOBAL_PROTECT(mentor_href_token)
 /proc/send_mentor_sound(client/C)
 	var/sound/pingsound = sound('yogstation/sound/misc/bikehorn_alert.ogg')
 	pingsound.volume = 90
-	if(GLOB.mentornoot)
+	if(GLOB.mentornoot || prob(1))
```
Comment (JamieD1) [inline yogstation/code/modules/mentor/mentor.dm]:
```suggestion
	if(GLOB.mentornoot || prob(10))
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**